deploy/libproject/deploy.rb

```diff
@@ -97,7 +97,7 @@ def setup_and_enter_docker(cmd_name, opts)
     exit 1
   end
 
-  key_file = Tempfile.new(["#{opts.account}-key", ".json"])
+  key_file = Tempfile.new(["#{opts.account}-key", ".json"], "/tmp")
```

Comment:
Thanks - could you add a comment above as well? By default Tempfile on OS X does not use a docker-friendly location.